eclass/frameworks.kde.org.eclass

```diff
@@ -25,6 +25,13 @@ esac
 if [[ -z ${_FRAMEWORKS_KDE_ORG_ECLASS} ]]; then
 _FRAMEWORKS_KDE_ORG_ECLASS=1
 
+# @ECLASS_VARIABLE: KDE_CATV
+# @DESCRIPTION:
+# Holds main Frameworks release number (major.minor) for use on same-category
+# dependencies.
+KDE_CATV=$(ver_cut 1-2)
```

Comment:
Would KF_CATV or KDE_KF_KCATV not be better?

Reply:
I chose `KDE_` as the informal `*.kde.org` eclass prefix. It should be just as generic as `KFMIN` and `QTMIN` already are, so there is no unnecessary potential for mixups.

Reply:
Oh, ok.

Reply:
Nah, I understand what you were going for now -- the eclass is solely for these packages, we might use this same variable name in other such cat-specific eclasses too. It doesn't make sense to allow it to be mixed or anything.